elrond-wasm-debug/Cargo.toml

```diff
@@ -15,7 +15,7 @@ categories = ["cryptography::cryptocurrencies", "development-tools::debugging"]
 
 [dependencies]
 elrond-wasm = { version = "0.7.0", path = "../elrond-wasm" }
```

Comment:
make sure you merge from master, where we have the latest version (0.7.1). We should also have mandos in the repo today, should make everything easier